### PR TITLE
Add the OpenSeadragon viewer for deep zoom images

### PR DESCRIPTION
**Pull request by gregvonkuster**

### CODE REVIEW

**Comment by dannon** [comment]
It looks like only `openseadragon.min.js` is used and we can drop the raw js?  Taking that a step further, seeing that it's available on npm, would you be up for swapping this over to the newer viz packaging?

**Comment by gregvonkuster** [comment]
@dannon Sure, but I am unaware of this newer viz packaging.  Can you point me to an example?

**Comment by dannon** [comment]
Sure, any of the ones that are in this 'PLUGIN_BUILD_IDS' list here:  https://github.com/galaxyproject/galaxy/blob/dev/client/gulpfile.js#L13

I think the media_player may be the one I touched last, so might be the best recent example?

**Comment by gregvonkuster** [comment]
I'm hoping I got it correct @dannon - please let me know if not.  Thanks!

**Comment by mvdbeek** [comment]
Just as a note for whoever is going to merge this, the JS files need to be rebased out of the history before merging.

**Comment by dannon** [comment]
Sorry for the delay, just managed to circle back to this.  Taking a look now and I'll get it in.

**Comment by dannon** [comment]
Should be good to go, this is working nicely in my hands.  I tweaked the deployment bits somewhat to make the `build` package script faster, and also included the nav images so those show up now.  @gregvonkuster Make sure this looks good to you still and I'm happy to merge it.

**Comment by gregvonkuster** [comment]
This looks great to me @dannon - thanks so much for your help on this!

**Comment by dannon** [comment]
Anytime, thanks for the viz!

File: config/plugins/visualizations/openseadragon/config/openseadragon.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE visualization SYSTEM "../../visualization.dtd">
+<visualization name="OpenSeadragon">
+    <description>Viewer for high-resolution zoomable images.</description>
+    <data_sources>
+        <data_source>
+            <model_class>HistoryDatasetAssociation</model_class>
+            <test type="isinstance" test_attr="datatype" result_type="datatype">images.Jpg</test>
+            <test type="isinstance" test_attr="datatype" result_type="datatype">images.Png</test>
```
Comment (bgruening) [inline config/plugins/visualizations/openseadragon/config/openseadragon.xml:9]: @gregvonkuster do you know if this also works for tiff?

Comment (gregvonkuster) [reply config/plugins/visualizations/openseadragon/config/openseadragon.xml:9]: I think read somewhere in the documentation that only jpg and png are currently supported simple image formats, but I cannot find it now.  Here is the page describing this  https://openseadragon.github.io/examples/tilesource-image/.  Perhaps a regular tiff image would work.  I don't remember if I tried loading one, but I'll do so.

I don't believe it works for pyramidal tiff images - https://github.com/openseadragon/openseadragon/issues/349, it seems that you have to convert them to dzi format - https://github.com/openseadragon/openseadragon/issues/351.  Animated gifs are also not supported https://ove.readthedocs.io/en/stable/ove-apps/packages/ove-app-images/README.html.

**Pull request merged**